charts/external-dns/values.yaml

```
@@ -49,6 +49,10 @@ securityContext:
   capabilities:
     drop: ["ALL"]
 
+# Defaults to `ClusterFirst`.
+# Valid values are: `ClusterFirstWithHostNet`, `ClusterFirst`, `Default` or `None`.
+dnsPolicy: ClusterFirst
```

```suggestion
dnsPolicy:
```
It's best not to specify a value and use the default if possible.